WooCommerce new documentation (#1299)
mercadopago/devsite-docs: guides/plugins/woocommerce/previous-requirements.en.md

# Prerequisites

To use the Mercado Pago integration with WooCommerce in your WordPress, you must meet the requirements below.

| Requirements | Description | Specifications |
|---|---|---|

> | Mercado Pago Seller account | To make sales, you need a seller account in Mercado Pago. If you don't have one, [click here](https://www.mercadopago.com.br/hub/registration/landing) to create. | Seller account on Mercado Pago|

Reviewer: ```suggestion
| Mercado Pago Seller account | To make sales, you need a seller account in Mercado Pago. If you don't have one, [click here](https://www.mercadopago[FAKER][URL][DOMAIN]/hub/registration/landing) to create. | Seller account on Mercado Pago|
```

> | WordPress | Online tool for stores, websites and blog creation. [Click here](https://br.wordpress.org/about/) for more information. | Installation version required 4.9.10 or higher. Tested up to 5.8.x |

Reviewer 2: ```suggestion
| WordPress | Online tool for stores, websites, and blog creation. [Click here](https://br.wordpress.org/about/) for more information. | Installation version required 4.9.10 or higher. Tested up to 5.8.x |
```

> | WooCommerce | WordPress Plugin that allows the creation of virtual stores from open source. [Click here](https://woocommerce.com/pt-br/woocommerce-features/) for more information. | Required 3.x or higher. Tested up to 5.6.x |

Author: ```suggestion
| WooCommerce | WordPress Plugin that allows the creation of virtual stores from open source. [Click here](https://woocommerce.com/woocommerce-features/) for more information. | Required 3.x or higher. Tested up to 5.6.x |
```

> | Environment | Hosting service. | MySQL, PHP or equivalent service that supports WordPress installation. |

Reviewer 2: ```suggestion
| Environment | Hosting service. | MySQL, PHP, or equivalent service that supports WordPress installation. |
```

> | Operational system | System responsible for managing the computer's hardware. | Linux x86, Windows x86-64 |

Reviewer 2: ```suggestion
| Operational system | System that is responsible for managing the computer's hardware. | Linux x86, Windows x86-64 |
```

| Web Server | Software responsible for accepting HTTP requests from clients, usually browsers, and serving them with HTTP responses. | Apache 2.x, Nginx 1.7.x |

> | PHP Version | PHP is a widely used programming language for web application development. For more information, [click here](https://www.php.net/). | PHP 5.6 and 7.x |

Reviewer 2: ```suggestion
| PHP Version | PHP is a widely-used programming language for web application development. For more information, [click here](https://www.php.net/). | PHP 5.6 and 7.x |
```

> | Database | Sets of files related to each other with records about people, places or things. | MySQL 5.6 or higher (Oracle or Percona), MariaDB 10.0 or higher.|

Reviewer 2: ```suggestion
| Database | Sets of files related to each other with records about people, places, or things. | MySQL 5.6 or higher (Oracle or Percona), MariaDB 10.0 or higher.|
```

| Extension Dependency | Extensions give new skills to PHP, adding more features. | PDO_MySQL, simplexml, mcrypt, hash, GD, DOM, iconv, curl, SOAP (for Webservices API). |

> | Additional configuration | Recommended settings for better performance and proper functioning of Wordpress, WooCommerce and Mercado Pago plugin. | [safe_mode](https://wordpress.org/plugins/safe-mode/) off* [memory_limit](https://docs.woocommerce.com/document/increasing-the-wordpress-memory-limit/) greater than 256MB (512MB recommended). |

Reviewer 2: ```suggestion
| Additional configuration | Recommended settings for better performance and proper functioning of WordPress, WooCommerce, and Mercado Pago plugin. | [safe_mode](https://wordpress.org/plugins/safe-mode/) off* [memory_limit](https://docs.woocommerce.com/document/increasing-the-wordpress-memory-limit/) greater than 256MB (512MB recommended). |
```

| SSL | Protocol that allows you to establish secure communications over the Internet for activities such as browsing, email, and other data transfers. | SSL Certificate. |

> | Backup copy **(optional)**| Copy all your information in order to guarantee a version without any changes if necessary. | We advise you to back up the online store before making any changes. When copying is complete, delete all files related to the previous version of the module. |

Reviewer 2: ```suggestion
| Backup copy **(optional)**| Copy all your information to guarantee a version without any changes if necessary. | We advise you to back up the online store before making any changes. When copying is complete, delete all files related to the previous version of the module. |
```

> | Backup copy **(optional)**| Copy all your information to guarantee a version without any changes if necessary. | We advise you to back up the online store before making any changes. When copying is complete, delete all files related to the previous version of the module. |

Author: ```suggestion
| Backup copy **(recommended)**| Copy all your information to guarantee a version without any changes if necessary. | We advise you to back up the online store before making any changes. When copying is complete, delete all files related to the previous version of the module. |
```

If all prerequisites are met, you can install the plugin.

> LEFT_BUTTON_REQUIRED_EN
>
> Plugin installation
>
> Follow the steps below to install the Mercado Pago plugin with WooCommerce.
>

> > [Plugin installation](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/plugins/woocommerce/installation)

Author: ```suggestion
> [Plugin installation](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/plugins/woocommerce/instalation)
```
